controllers/ibmpowervsmachine_controller.go

```diff
@@ -198,6 +198,23 @@ func (r *IBMPowerVSMachineReconciler) getOrCreate(scope *scope.PowerVSMachineSco
 	return instance, err
 }
 
+// handleLoadBalancerPoolMemberConfiguration handles loadbalancer pool member creation flow.
+func (r *IBMPowerVSMachineReconciler) handleLoadBalancerPoolMemberConfiguration(machineScope *scope.PowerVSMachineScope) (ctrl.Result, error) {
+	if !util.IsControlPlaneMachine(machineScope.Machine) {
+		return ctrl.Result{}, nil
+	}
+	machineScope.Info("Configuring control plane machine to backend LoadBalancer pool", "machine name", machineScope.IBMPowerVSMachine.Name)
```

Comment:
I think its better use single name for key in logs, like `machineName` instead of `machine name`, It might be easy to filter out in observability [tools](https://cluster-api.sigs.k8s.io/developer/logging#example-queries).

Reply:
Done